int32[] ids
std_msgs/Header header
geometry_msgs/Pose[] poses
